### ISSUE TITLE
suggestion - there should be some consequences for breaking into the pharmacy backroom and stealing all the meds

### ISSUE THREAD

**Issue by pinkamedic** [loiste-interactive/Obenseuer-Issues#2639]
with how things are right now you can break into the backroom steal a ton of meds and more or less be set on all needs for meds (yetleast for painkiller, sleeping, anti depressants) there really should be some draw back from breaking into the backroom, maybe a large wanted level or at max being banned from getting prescription meds as the women will refuse to service you

**Comment by Nextej** [comment]
Yeah, it's weird the NPCs there don't react to some random guy suddenly having an employee keycard and entering restricted areas, realistically they would immediately start question you and probably alert nearby police officer. Perhaps you should have a lab coat worn on you that you can find only from looting Ten A labs.
